playbooks/archivematica-centos7/singlenode.yml

```diff
@@ -7,12 +7,41 @@
       tags:
         - "always"
 
-    - name: "Set SELinux to Permissive"
-      command: "setenforce Permissive"      
-      become: "yes"
+   - name: "Selinux boolean 1"
```

Comment:
Could this be more descriptive?
e.g. `SELinux: allow HTTPD scripts and modules to connect to the network.`

Same with the others.